Packs/SpurContextAPI/Integrations/SpurContextAPI/SpurContextAPI.py

```diff
@@ -127,11 +127,10 @@ def main() -> None:
     :rtype:
     """
 
-    api_key = demisto.params().get('credentials', {}).get('password')
-    base_url = "https://api.spur.us/"
+    api_key = demisto.params().get('credentials', {}).get('password') or demisto.params().get('credentials')
```

Comment:
that's not correct, type 9 is always a dict with keys identifier and password,

Reply:
Can you help here please? I added the **or demisto.params().get('credentials')** because by default the integration were not able to find the password, only when switched to credentials method.

Reply:
type 9 is always a dict of 2 keys, identifier and password, i dont know how you got a result for demisto.params().get('credentials'), maybe it returned a result for the first part of the condition,
you can run any command with debug-mode=true, and see in the logs the params, and how are they defined.
what i know for sure is, it's not possible to get a value from type 9